spec/amp-story-player.md

```diff
@@ -126,3 +130,192 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player. These methods are exposed on the DOM HTML element, `const playerEl = document.querySelector('amp-story-player')` and on the global class variable,`const player = new AmpStoryPlayer(window, playerEl)`.
+
+### Methods
+
+#### load
+
+Will initialize the player manually. This can be useful when the player is dynamically added such as when using frameworks like React.
```

Comment:
I think our documentation should be framework agnostic. At the scale of the web not that many people use React, and we don't want to update our documentation if there's a React update.